### PR TITLE
Add guidance around the number of sources a user should create

### PR DESCRIPTION
**Pull request by sarahrudy**
Add info block about whether customer should set up one source or multiple sources

### Proposed changes

Add info block about whether customer should set up one source or multiple sources

### Merge timing
- ASAP once approved?

### CODE REVIEW

File: src/connections/sources/index.md [hunk 1]
```diff
@@ -71,6 +71,13 @@ To create a source:
 4. Enter a name for your source as well as any information on the setup page.
 5. Click **Add Source**.
 
+> info ""
+> **One source or multiple sources?**
```
Comment (markzegarelli) [inline src/connections/sources/index.md:75]:
```suggestion
```

File: src/connections/sources/index.md [hunk 2]
```diff
@@ -71,6 +71,13 @@ To create a source:
 4. Enter a name for your source as well as any information on the setup page.
 5. Click **Add Source**.
 
+> info ""
```
Comment (markzegarelli) [inline src/connections/sources/index.md:74]:
```suggestion
> info "One source or multiple sources?"
```

File: src/connections/sources/index.md [hunk 3]
```diff
@@ -71,6 +71,13 @@ To create a source:
...
+> **One source or multiple sources?** 
+> Ultimately, it's up to you, but it really depends on how you want to organize your data. We suggest that you create one source for each type of data you want to collect. For example, a customer might have one source for all of their website tracking and a different source for any mobile tracking. We typically recommend having one source for each type of data (different libraries) for the following reasons:
```
Comment (markzegarelli) [inline src/connections/sources/index.md:76]:
```suggestion
> Segment suggests that you create one source for each type of data you want to collect. For example, a you might have one source for all of their website tracking and a different source for any mobile tracking. Creating one source per data type provides the following benefits:
```

Comment (markzegarelli) [reply src/connections/sources/index.md:76]: Important to remember that we're our customers and prospects are the main audience for our docs. We typically just use `you` as opposed to `a customer`.

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: src/connections/sources/index.md (hunk 3 above)
Comment (sarahrudy) [reply src/connections/sources/index.md:75]: Thank you for pointing that out!

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**